src/drivers/riscv_cpu.c

```diff
@@ -636,7 +636,8 @@ int __metal_driver_cpu_exception_register(struct metal_cpu *cpu, int ecode,
 int  __metal_driver_cpu_get_instruction_length(struct metal_cpu *cpu, uintptr_t epc)
 {
     /* Per ISA compressed instruction has last two bits of opcode set */
```

Comment:
Probably a good idea to update this comment to describe better what we're checking for.

Reply:
Yes, let me add more description for it.